Update episode 02 licensing (#3)
esciencecenter-digital-skills/lesson-FAIR-software: _episodes/02-licensing.md

- "When is a license important?"
- "How to choose a license?"
objectives:

> - "Learn about software license"

Reviewer: ```suggestion
- "Learn about software licenses"
```

- "Add a license to a GitHub repository"
keypoints:

> - "It is important to add a license to your open-source code."

Reviewer: Actually, it is not open source, when there is no license. Maybe 'It is important to add an open-source license to make your code open'.

Author: Done!

- "There are several common licenses that have different permissiveness."
---

> ## [Recommendation 2](https://fair-software.eu/recommendations/license)
>
> Add a license.
{: .callout}
![recommendation2]({{ page.root }}/fig/recommendation2.png)

## Why a license is important

Any creative work (including software) is automatically protected by copyright.
Even when the software is available via code sharing platforms such as GitHub,
no one can use it unless they are explicitly granted permission.
This is done by adding a software license, which defines the set of rules
and conditions for people who want to use the software.

Be aware that you, as the developer of a given piece of software,

> may not be a copyright holder of the code you write. Usually, the copyright

Reviewer: ```suggestion
may not be the copyright holder of the code you write. Often, the copyright
```

holder of the work is the employer (or hiring party) and not the author of the work.

## Choose a license

> We recommend you stick to one of the more popular licenses. Because these

Reviewer: ```suggestion
We recommend you use one of the more popular licenses. Because these
```

are typically written by lawyers, the license text is precise in expressing
its terms. While that carries the unfortunate side effect of being difficult
to understand, the widespread use of the more popular licenses means that
there is a larger number of people who understand how the letter of
the law should be interpreted.

Some commonly used licenses are the ``Apache-2.0`` and ``MIT`` licenses.
These permissive licenses have very few restrictions, allowing others to
easily reuse your work.

There are several websites that can help you choose a license:

- [choosealicense.com](https://choosealicense.com/): helps to choose an open-source license.
- [tldrlegal.com](https://tldrlegal.com/): summarizes what is allowed and not allowed under a given license.

> ## Choosing a license for a project at the university
>
> Suppose you write code for a project at the university.

> > Open sourcing software is not yet best practice in your community,

Reviewer: ```suggestion
> Open sourcing software may not yet be a common practice in your community,
```
